.github/workflows/ci.yml

```
@@ -35,14 +42,16 @@ jobs:
         if: "${{ env.DOCKER_USERNAME != '' }}"
         uses: docker/metadata-action@v4
         with:
-          images: ${{ secrets.DOCKER_USERNAME }}/jellyplex-watched # list of Docker images to use as base name for tags
+          images: |
+            ${{ secrets.DOCKER_USERNAME }}/jellyplex-watched
+            ghcr.io/${{ secrets.DOCKER_USERNAME }}/jellyplex-watched

```

I think this might work
```suggestion
        uses: docker/metadata-action@v4
        with:
          images: |
            ${{ secrets.DOCKER_USERNAME }}/jellyplex-watched,enable={{ secrets.DOCKER_USERNAME != '' }}
            ghcr.io/${{ secrets.DOCKER_USERNAME }}/jellyplex-watched

```